grafana-plugin/e2e-tests/integrations/heartbeat.test.ts

```diff
@@ -7,8 +7,7 @@ const HEARTBEAT_SETTINGS_FORM_TEST_ID = 'heartbeat-settings-form';
 
 test.describe("updating an integration's heartbeat interval works", async () => {
   const _openHeartbeatSettingsForm = async (page: Page) => {
-    await page.getByTestId('integration-settings-context-menu-wrapper').getByRole('img').click();
```

Comment:
I believe the role of the hamburger icon changed across various versions of grafana.. this is much more stable